certified-connectors/ZellisIntelligencePlatform/apiDefinition.swagger.json

```diff
@@ -0,0 +1,1063 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "title": "Zellis Intelligence Platform",
+    "description": "This connectors consists various actions you can use to perform your daily tasks.",
```

Comment:
@basha-k This description is generic one which can be improved. Please provide a detailed description that your connector specifically points to.